src/Entity/QuerySearch.php

```diff
@@ -0,0 +1,184 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EMS\CoreBundle\Entity;
+
+use Doctrine\Common\Collections\Collection;
+use Doctrine\ORM\Mapping as ORM;
+use Ramsey\Uuid\Doctrine\UuidGenerator;
+use Ramsey\Uuid\Uuid;
+use Ramsey\Uuid\UuidInterface;
+
+/**
+ * @ORM\Table(name="query_search")
+ * @ORM\Entity()
+ * @ORM\HasLifecycleCallbacks()
+ */
+class QuerySearch implements EntityInterface
+{
+    /**
+     * @ORM\Id
+     * @ORM\Column(type="uuid", unique=true)
+     * @ORM\GeneratedValue(strategy="CUSTOM")
+     * @ORM\CustomIdGenerator(class=UuidGenerator::class)
+     */
+    private UuidInterface $id;
+
+    /**
+     * @ORM\Column(name="created", type="datetime")
+     */
+    private \Datetime $created;
+
+    /**
+     * @ORM\Column(name="modified", type="datetime")
+     */
+    private \Datetime $modified;
+
+    /**
+     * @ORM\Column(name="label", type="string", length=255)
+     */
+    private string $label;
+
+    /**
+     * @ORM\Column(name="name", type="string", length=255)
+     */
+    private string $name;
+
+    /**
+     * @var Collection <int,Environment>
+     *
+     * @ORM\ManyToMany(targetEntity="Environment", cascade={"persist"})
+     * @ORM\JoinTable(name="environment_query_search",
+     *      joinColumns={@ORM\JoinColumn(name="query_search_id", referencedColumnName="id")},
+     *      inverseJoinColumns={@ORM\JoinColumn(name="environment_id", referencedColumnName="id")}
+     *      )
+     */
+    protected Collection $environments;
+
+    /**
+     * @var array<string, mixed>
+     *
+     * @ORM\Column(name="options", type="json", nullable=true)
+     */
+    private array $options;
+
+    /**
+     * @ORM\Column(name="order_key", type="integer")
+     */
+    private int $orderKey;
+
+    public function __construct()
+    {
+        $now = new \DateTime();
+        $this->id = Uuid::uuid4();
+        $this->created = $now;
+        $this->modified = $now;
+        $this->environments = new \Doctrine\Common\Collections\ArrayCollection();
+        $this->options = [
+            'query' => '{}',
+        ];
+    }
```

Comment:
Member attribut that are not nullable must be assigned to something  or in the constructor or  with the declration:
```suggestion
    private int $orderKey = 9999;

    public function __construct()
    {
        $now = new \DateTime();
        $this->id = Uuid::uuid4();
        $this->created = $now;
        $this->modified = $now;
        $this->environments = new \Doctrine\Common\Collections\ArrayCollection();
        $this->options = [
            'query' => '{}',
        ];
    }
```